scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -130,10 +130,103 @@ def __init__(self):
+
+    ## Event Collection Leader
+
+    The Scalyr Agent uses a simple leader election algorithm to decide which Scalyr Agent should retrieve the cluster's Events and upload them to Scalyr. This is necessary because the Events pertain to the entire cluster (and not necessarily just one Node). Duplicate information would be uploaded if each Scalyr Agent Pod retrieved the cluster's Events and uploaded them.
+
+    By default, the leader election algorithm selects the Scalyr Agent Pod running on the oldest Node, excluding the Node running the master. To determine which is the oldest node, each Scalyr Agent Pod will retrieve the entire cluster Node list at start up, and then query the API Server once every `leader_check_interval` seconds (defaults to 60 seconds) to ensure the leader is still alive. For large clusters, performing these checks can place noticeable load on the K8s API server. There are two ways to address this issue, with different impacts on load and flexibility.
+
+    ### Node Labels
+
+    The first approach is to add the label `agent.config.scalyr.com/events_leader_candidate=true` to any node that you wish to be eligible to become the events collector.  This can be done with the following command:
+
+    `kubectl label node <nodename> agent.config.scalyr.com/events_leader_candidate=true`
+
+    Where `<nodename>` is the name of the node in question.  You can set this label on multiple nodes, and the agent will use the oldest of them as the event collector leader.  If the leader node is shutdown, it will fallback to the next oldest node and so on.
+
+    Once the labels have been set, you also need to configure the agent to query these labels.  This is done via the `check_labels` configuration option in the Kubernetes Events monitor config (which is off by default):
+
+    ```
+    ...
+    monitors: [
+      {
+        module: "scalyr_agent.builtin_monitors.kubernetes_events_monitor",
+        ...
+        check_labels: true
+      }
+    ]
+    ...
+    ```
+
+    If `check_labels` is true then instead of querying the API server for all nodes on the cluster, the Scalyr agents will only query the API server for nodes with this label set.  In order to reduce load on the API server, it is recommended to only set this label on a small number of nodes.
+
+    This approach reduces the load placed on the API server and also provides a convenient fallback mechanism for when an exister event collector leader node is shutdown.
+
+    ### Hardcoded Event Collector
+
+    The second approach is to manually assign a leader node in the agent config.  This is done via the `leader_node` option of the Kubernetes Events monitor:
+
+    ```
+    ...
+    monitors: [
+      {
+        module: "scalyr_agent.builtin_monitors.kubernetes_events_monitor",
+        ...
+        leader_node: <name of leader node>
```

Comment:
Add quotes around the <name of leader node>

Reply:
Quote around the.......?

I guess <name of the leader node> but in case you also meant the keys, I added them around the keys also.